library/ceph_orch_apply.py

```diff
@@ -137,6 +140,14 @@ def run_module() -> None:
     expected = parse_spec(module.params.get('spec'))
     current_spec = retrieve_current_spec(module, expected)
 
+    # Remove dynamic attributes
+    if current_spec and 'status' in current_spec:
+        del current_spec["status"]
+    if current_spec and 'events' in current_spec:
+        del current_spec["events"]
```

Comment:
`current_spec` can't be something else than either an empty dict `{}` or a valid spec with values.

```suggestion
DYNAMIC_ATTRIBUTES: List[str] = ['status', 'events']
for attribute in DYNAMIC_ATTRIBUTES:
    if attribute in current_spec:
        del current_spec[attribute]
```